data-prepper-logstash-configuration/src/main/resources/org/opensearch/dataprepper/logstash/mapping/kv.mapping.yaml

```diff
@@ -0,0 +1,11 @@
+pluginName: key_value
```

Comment:
I would like to see the key_value processor conversion added to one of the integration tests.